### ISSUE TITLE
createcpgregions mode for cpg input files needed

### ISSUE THREAD

**Issue by avilella** [lidaof/methylQA#4]
Would it be possible to include the necessary code/binaries in the methylQA repo that would allow one to create the cpg.bed and genome.chrsizes.tsv input files needed to run methylqa bismark mode? Something like:

```
methylqa createcpgregions reference.fa
```

which would output:

```
reference.cpg.bed
reference.chrsizes.tsv
```

I know it's been mentioned that it uses code from the jkent library, so it would be good to have all this in the methylqa repo, so that from a methylqa git clone, one can create the input files needed for methylqa bismark using the methylqa repo exclusively.

**Comment by lidaof** [comment]
Good idea....should be not that difficult. I'll look into it. Thanks for suggestions.
